version/version.go

```diff
@@ -1,7 +1,7 @@
 package version
 
-// VERSION ...
-const VERSION = "0.10.4"
```

Comment:
Are you sure we do not need this version? I think there is a case where the plugin is downloaded from git (when binary release is not found)

Reply:
Nice, I didn't know about this feature. But I traced it back to the install code and it looks like when a plugin is installed from source, it's version is an empty string: https://github.com/bitrise-io/bitrise/blob/master/plugins/install.go#L276